### ISSUE TITLE
Wrapup report

### ISSUE THREAD

**Issue by mkutu** [precision-sustainable-ag/SemiF-PlantDetection#58]
Provide a report in confluence about this work. Sections include, but not limited to:

- problem (2-4 bullet points)
- data: explain our dataset and how it was pulled, refined, and used. Talk about data size, difference between our dataset and Soren's
- hyperparameters and model architecture (2-3 points)
- multiscale inferenincg (the "what" and "how")
- why we're overfitting (brief)
- results: comparing...
  - soren's original
  - training from scratch (new model)
  - training form Soren's checkpoint (old model)
  - addition of aug (verrsion 1)
  - addition of aggresive augment (version 2)
  - edge_aware
  - post_fusion_nms
- limitations: be fully transparent here
  - after the first aggressive augmentation training run, accuracy at higher IoU thresholds takes a considerable drop. There may be future work here. But this is ok for our situation because we inference at 0.55 which is where the models out perform...
